src/KeyringController.ts

```diff
@@ -184,7 +211,7 @@ export class KeyringController extends BaseController<BaseConfig, KeyringState>
 	 * @param password - Password of the keyring
 	 * @returns - Promise resolving to the seed phrase
 	 */
-	exportSeedPhrase(password: string): Promise<string> {
+	exportSeedPhrase(password: string) {
```

Comment:
Why did you remove the return type here?

Reply:
it doesn't return a `promise`, only a `string`